### PR TITLE
Instantclick.io

### PR DESCRIPTION
**Pull request by Sumukh**
Uses instantclick.io

### CODE REVIEW

File: server/templates/staff/base.html [hunk 1]
```diff
@@ -76,6 +76,8 @@
 
       {% assets "adminlte_js" %}
           <script type="text/javascript" src="{{ ASSET_URL }}"></script>
+          <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/instantclick/3.0.1/instantclick.min.js"></script>
```
Comment (Sumukh) [inline server/templates/staff/base.html]: Will be bundled into our assets js file later.

**Comment by knrafto** [comment]
Is there some JS library that's rendering the times? It makes the submissions table jump around when reloading (e.g. clicking a flag), and does weird things like this:
screen shot 2016-01-29 at 8 08 20 pm

**Comment by Sumukh** [comment]
Yes - that's moment.js. It's a hack for now until I implement a jinja filter to convert times to PST

**Comment by Sumukh** [comment]
This is now only active when debug mode is off. 
Filed an issue for the timestamps  (#667).

**Comment by knrafto** [comment]
I'm worried that this won't be well tested and it's easy to mess up - for example, if we miss blacklisting a logout link, hovering over the link will mysteriously log the user out. I think we should hold off on this for now and come back to it when the app is more mature

**Comment by alvinwan** [comment]
You could use Python's arrow for easier time conversions. Here is a sample you could probably just copy and paste with minor changes.
```
import arrow
utc = arrow.get(py_datetime)
utc.to('local')  # converts to US/Pacific time
utc.humanize() # for "3 hours ago"-like string
utc.format('YYYY-MM-DD HH:mm:ss')  # 2016-01-22 09:11:12
```
Arrow was, in fact, inspired by moment.js. http://crsmithdev.com/arrow/

**Comment by knrafto** [comment]
Logging out should use POST not GET

**Comment by Sumukh** [comment]
This fix is included in https://github.com/Cal-CS-61A-Staff/ok/tree/v3/sumukh/assorted

**Pull request closed without merging**